_posts/2021-11-07-canary.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: "Use a canary in your Apache Kafka coal mine"
```

Comment:
What about something like: `Why should you use a canary in your Apache Kafka coal mine?`

Should we capitalize `Canary` as it's the name of the component?